File: iOS/PrayerTimes/PrayerTimes/Time.swift
//
//  Time.swift
//  Prayer Times
//
//  Created by Farhan Yuda Pahlevi on 5/22/18.
//  Copyright © 2018 Apple Inc. All rights reserved.
//

import Foundation

class Time{
    var name:String?
    var time:String?
    var gregorian:String?
    var hijri:String?
    var timeStamp:String?
    var latitude:Double?
    var longitude:Double?
    var timeZone:String?
    var method:String?
    
    init?(name:String, time:String, gregorian:String, hijri:String, timeStamp:String, latitude:Double, longitude:Double, timeZone:String, method:String){
        self.name = name
        self.time = time
        self.gregorian = gregorian
        self.hijri = hijri
        self.timeStamp = timeStamp
        self.latitude = latitude
        self.longitude = longitude
        self.timeStamp = timeZone
        self.method = method
    }
}

File: Android/PrayerTimes/app/src/main/java/com/deggan/prayertimes/Time.kt
package com.deggan.prayertimes

/**
 * Created by farhan on 5/22/18.
 */
data class Time(val fajr: String,
                val sunrise: String,
                val dhuhr: String,
                val asr: String,
                val sunset: String,
                val maghrib: String,
                val isha: String,
                val imsak: String,
                val midnight: String,
                val gregorian: String,
                val hijri: String,
                val timeStamp: String,
                val latitude: Double,
                val longitude: Double,
                val timeZone: String,
                val method: String)